contracts/root/StakeManagerInterface.sol

```diff
@@ -3,11 +3,11 @@ pragma solidity ^0.4.24;
 
 // ERC900
 interface StakeManagerInterface {
-  event Staked(address indexed user, uint256 amount, uint256 total, bytes data);
+  event Staked(address indexed user, address indexed signer, uint256 indexed activatonEpoch, uint256 amount, uint256 total,  bytes data);
```

Comment:
Do we need `bytes data`?

Reply:
not immediately, we might need it in the future.